### ISSUE TITLE
What class to band or other artistic group?

### ISSUE THREAD

**Issue by TommiRTVA** [ebu/ccdm#219]
![kuva](https://user-images.githubusercontent.com/47444068/198557843-78add725-ea64-4626-a04f-4f7877a91977.png)

Would a band or other artistic group be just Agent?

Best regards,

Tommi Lehtonen

**Comment by tormodv** [comment]
In my mind, a band is just an organisation. 
Large bands like Oslo Philharmonic are registered as and work like an organisation. 

Smaller groups that play together will be comparable to the larger ones. 

Best regards 

Tormod

**Comment by JuergenGrupp** [comment]
I agree with Tormod. How about adding a class "ArtisticGroup" as subclass to "Organisation" to the standard? It could represent orchestras, music bands, dance groups, artist collectives, ...

**Comment by TommiRTVA** [comment]
I think class ArtisticGroup would make good distinction from companies and others alike.

best regards,

Tommi

**Comment by kimviljanen** [comment]
Just a thought: Being ”artistic” is perhaps difficult to define - by definition art can be anything. So any organization could be more or less also artistic by nature. How can we define which groups are artistic and which are not? For example, a broadcasting company or movie production company might consider itself to be artistic. For example the music band Queen or Madonna might consider them to be an artistic group or individual, but they are very much about business and brand management too. Hence, it might not make sense to create a separate subclass for ArtisticGroup but consider artistic as a goal or vision of an organization, such as a music band (organization) or movie production company. 

I chose Madonna as one example, because it highlights one challenge: some artistic entities are individuals (e.g. Madonna), so defining a person to be a group is somewhat odd.

So all in all, both individuals and organisations may produce artistic outcomes or define themselves to be artistic. I’m not sure if that however means that we should reserve a class for them.

**Comment by TommiRTVA** [comment]
@kimviljanen makes a good point. Maybe something like "genre" or other definition/type could be added to organisation.

**Comment by tormodv** [comment]
It is already a `:objectType only skos:Concept` on the class in question, so it is possible to make a list of different organizations expressed as skos concepts.
